client/webserver/site/src/js/wallets.ts

```diff
@@ -815,26 +816,39 @@ export default class WalletsPage extends BasePage {
     }
     let firstOther = false
     Doc.empty(page.balanceDetailBox)
-    const addSubBalance = (category: string, subBalance: number) => {
+    const addSubBalance = (category: string, subBalance: number, tooltipMsg: string) => {
       const row = page.balanceDetailRow.cloneNode(true) as PageElement
       if (firstOther) {
         row.classList.add('first-other')
         firstOther = false
       }
       page.balanceDetailBox.appendChild(row)
       const tmpl = Doc.parseTemplate(row)
-      tmpl.category.textContent = category
+      if (tmpl.category.firstElementChild) tmpl.category.firstElementChild.textContent = category
+      else tmpl.category.textContent = category
+      if (tooltipMsg) {
+        tmpl.tooltipMsg.dataset.tooltip = tooltipMsg
+        Doc.show(tmpl.tooltipMsg)
+      }
       tmpl.subBalance.textContent = Doc.formatCoinValue(subBalance, ui)
     }
-    addSubBalance('Available', bal.available)
-    addSubBalance('Locked', totalLocked)
-    addSubBalance('Immature', bal.immature)
-    const sortedCats = Object.entries(bal.other || {})
-    sortedCats.sort((a: [string, number], b: [string, number]): number => a[0].localeCompare(b[0]))
+    addSubBalance(intl.prep(intl.ID_AVAILABLE_C), bal.available, '')
+    addSubBalance(intl.prep(intl.ID_LOCKED_C), totalLocked, intl.prep(intl.ID_LOCKED_BAL_MSG))
+    addSubBalance(intl.prep(intl.ID_IMMATURE_C), bal.immature, intl.prep(intl.ID_IMMATURE_BAL_MSG))
+    const sortedBalCats = Object.entries(bal.other || {})
+    sortedBalCats.sort((a: [string, CustomBalance], b: [string, any]): number => a[0].localeCompare(b[0]))
     firstOther = true
-    if (bal.contractlocked > 0) addSubBalance('Swapping (locked)', bal.contractlocked)
-    if (bal.bondlocked > 0) addSubBalance('Bonded (locked)', bal.bondlocked)
-    for (const [cat, sub] of sortedCats) addSubBalance(cat, sub)
+    const lockedBal = (category: string): string => {
+      return category + ' (' + intl.prep(intl.ID_LOCKED) + ') '
+    }
+    if (bal.contractlocked > 0) addSubBalance(lockedBal(intl.prep(intl.ID_SWAPPING)), bal.contractlocked, intl.prep(intl.ID_LOCKED_SWAPPING_BAL_MSG))
+    if (bal.bondlocked > 0) addSubBalance(lockedBal(intl.prep(intl.ID_BONDED)), bal.bondlocked, intl.prep(intl.ID_LOCKED_BOND_BAL_MSG))
+    for (const [cat, bal] of sortedBalCats) {
```

Comment:
We need to display `orderLocked` as well. Not sure why we weren't already doing that. 

I also think we should reorganize the display so that the locked sub-balances show as subcategories of the locked total, e.g. roughly
![image](https://github.com/decred/dcrdex/assets/6109680/9f8e20ba-3020-4978-b5f4-0f2d0e2d83e4)
But this can be done in a followup.

Reply:
> I also think we should reorganize the display so that the locked sub-balances show as subcategories of the locked total, e.g. roughly

Okay.

Reply:
> I also think we should reorganize the display so that the locked sub-balances show as subcategories of the locked total, e.g. roughly

@buck54321, From the comment for these balance fields, It seems they are not included in `asset.Balance.Locked` and might be misleading to include them as a subcategory of the locked total, or I'm I missing something?


https://github.com/decred/dcrdex/blob/4a2ba0ff4606c81b7c4918c84ef8927756eeea10/client/core/types.go#L111-L118